website/content/docs/concepts/resource-quotas.mdx

```diff
@@ -20,24 +20,29 @@ developers.
 
 Vault provides a feature, resource quotas, that allows Vault operators to specify
 limits on resources used in Vault. Specifically, Vault allows operators to create
-and configure API rate limits.
+and configure API rate limits. Users of Vault Enterprise have a second option, alongside
+rate limits, [lease-count quotas](/docs/enterprise/lease-count-quotas), which can
+limit the number of leases that can be in use at one time.
 
 ## Rate Limit Quotas
 
 Vault allows operators to create rate limit quotas which enforce API rate
 limiting using a [token bucket](https://en.wikipedia.org/wiki/Token_bucket) algorithm.
-A rate limit quota can be created at the root level or defined on a namespace or
-mount by specifying a `path` when creating the quota. The rate limiter is applied
-to each unique client IP address on a per-node basis (i.e. rate limit quotas
-are not replicated). A client may invoke `rate` requests at any given second,
+A rate limit quota can be created at the root level or defined on a namespace,
+mount, or full API path by specifying a `path` when creating the quota. The rate
+limiter is applied to each unique client IP address on a per-node basis (i.e. rate limit
+quotas are not replicated). A client may invoke `rate` requests at any given second,
 after which they may invoke additional requests at `rate` per-second.
 
 A rate limit quota defined at the root level (i.e. empty `path`) is inherited by
 all namespaces and mounts. It acts as a single rate limiter for the entire Vault
 API. A rate limit quota defined on a namespace takes precedence over the global
-rate limit quota, and a rate limit quota defined for a mount takes precedence over
-the global and namespace rate limit quotas. In other words, the most specific
-quota rule will be applied.
+rate limit quota, a rate limit quota defined for a mount takes precedence over
+the global and namespace rate limit quotas, and a rate limit quota defined for
+a specific path will take precedence over global, namespace, and mount quotas.
```

Comment:
I'm wondering if the precedence ordering would be easier to consume as a numbered list. Not at all a requirement but might be easier to identify when skimming through logs if folks are trying to relocate that information.